coach/resources/styles/components/graphics/launcher-animation.less

```diff
@@ -0,0 +1,247 @@
+}
+
+.keyframes(launcher-answer-enter; {
+    0% { opacity: 0; .translate(-200px; 0); }
+   20% { opacity: 0; }
+   60% { opacity: 1; .translate(30px; 0); }
+   80% {             .translate(-10px; 0); }
+  100% {             .translate(0; 0); }
+});
+
+
+.keyframes(launcher-mug-leave; {
+    0% { opacity: 1; .translate(0; 0); }
+   20% { opacity: 1; .translate(-20px; 0); }
+   40% { opacity: 0; .translate(600px; 0); }
+  100% { opacity: 0; .translate(600px; 0); }
+});
+
+.launcher-concepts-animate() {
+  &.launcher-question { .animation-duration(100ms); }
+  &.launcher-a { .animation-duration(200ms); }
+  &.launcher-b { .animation-duration(300ms); }
+  &.launcher-c { .animation-duration(400ms); }
+  &.launcher-d { .animation-duration(500ms); }
+};
+
+.launcher-base() {
+  position: relative;
+  width: 100%;
+
+  text-align: center;
+  background: @openstax-neutral-lightest;
+
+  .laptop-and-mug {
+    width: 100%;
+    position: absolute;
+    left: 0;
+  }
+
+  .btn {
+    position: relative;
+    top: 5%;
+  }
+
+  .launcher-answer {
+    .launcher-concept();
+  }
+  .launcher-desk {
+    .transition(all 200ms linear);
+  }
+  .launcher-concept-coach {
+    .transform-origin(50% 25%);
+    .transition(all 400ms ease-in);
+  }
+  .btn,
+  .launcher-concept-coach-shine-bottom,
+  .launcher-question .launcher-text polygon,
+  .launcher-section-label {
+    .transition(opacity 200ms linear);
+  }
+  .launcher-concept-row {
+    .transition(opacity 800ms linear);
+  }
+  .launcher-background,
+  .launcher-concept-coach-screen {
+    .transition(fill 200ms linear);
+  }
+}
+
+.launcher-animate() {
+
+  .launcher-a { .launcher-concept(selected, 2600ms); }
+  .launcher-b { .launcher-concept(selected, 4600ms); }
+  .launcher-c { .launcher-concept(selected, 600ms); }
+  .launcher-d { .launcher-concept(selected, 6600ms); }
+
+}
+
+.launcher-animate(active) {
+  .launcher-concept-row {
+    .launcher-concepts-animate();
+    .animation(launcher-answer-enter ease-in);
+  }
+
+  .btn {
+    .box-shadow(0 1px 6px rgba(0, 0, 0, 0.5));
+  }
+}
+
+.launcher-animate(launching) {
+  .btn,
+  .launcher-concept-coach-shine-bottom,
+  .launcher-question .launcher-text polygon,
+  .launcher-section-label,
+  .launcher-concept-row {
+    opacity: 0;
+  }
+
+  .launcher-desk {
+    .translate(0, 100%);
+    .transform-origin(50% 100%);
+  }
+  .launcher-concept-coach {
+    .transform(~'translate(-62px, 30px) scale(3.97)');
+  }
+  .launcher-answer {
+    .launcher-concept(launching);
+  }
+  .launcher-coffee {
+    .animation(launcher-mug-leave 300ms ease-in);
+    .animation-fill-mode(both);
+  }
+  .launcher-concept-coach-screen {
+    fill: @openstax-white;
+  }
+  .launcher-background {
+    fill: @openstax-neutral-lighter;
+  }
+  .launcher-laptop {
+    opacity: 0;
+  }
+}
+
+.launcher-animate(closing) {
+  .launcher-concept-row {
+    .translate(0; 0);
+  }
+  .launcher-concept-coach {
+    .scale(1);
+  }
+  .launcher-coffee {
+    .translate(0; 0);
+  }
+}
```

Comment:
Haha I shoulda read the entire change before commenting.   Plz disregard the previous